### ISSUE TITLE
404 BrowserSync

### ISSUE THREAD

**Issue by Ti-webdev** [typicode/hotel#91]
Would hotel proxy to other ports for BrowserSync, Livereload, etc
Now it is return 404

**Comment by typicode** [comment]
I'm not familiar with BrowserSync and Livereload.
Can you explain what the problem is (I'll try to reproduce later)?

But I don't see reasons why it wouldn't be possible to use it with them. It should be noted that it's possible to set a fixed PORT if your app expect to be run on a particular one.
For example: `hotel add -p 3000 'cmd $PORT'`

**Comment by Ti-webdev** [comment]
I have a different apps which use two ports (for example Ionic Framework). First port used for deliver web to browser, second for live reload.

If I open it through http://127.0.0.1:%$PORT that works fine.

Now I open it with proxy, like http://site.dev.
In this case BrowserSync sent requests to http://site.dev:3000 and hotel proxy returned 404 for it.

**Comment by typicode** [comment]
OK, I see. Thank you for the explanations.
What's the command you're using to start browser-sync so that I can try locally?

**Comment by typicode** [comment]
I did a quick test, using the following command and it seems to work fine.
Let me know if it solves the issue for you.

```sh
# created an index.html file in myapp
# there's just a <body> with some text in it.
myapp $ hotel add 'browser-sync start --server --port $PORT'
```

In case you've added Browsersync script in `index.html`:

```js
<script id="__bs_script__">//<![CDATA[
    document.write("<script async src='http://HOST:3000/browser-sync/browser-sync-client.2.12.5.js'><\/script>".replace("HOST", location.hostname));
//]]></script>
```

Try Removing it, as `browser-sync start --server --port $PORT` automatically adds it.

**Comment by Ti-webdev** [comment]
It's okay in this case.

I tried use hexo-browsersync with it. I can't set port from env :(
As solution proxy.pac can ignore other ports.

**Comment by alqaline94** [comment]
I tried using browser sync with this command :

`browser-sync start --server . --files . --port %PORT%`

and it works. Leave the quotes for the command arguments

**Comment by typicode** [comment]
Closing this as it might be resolved by `v0.5.13` (see https://github.com/typicode/hotel/issues/153)

> In this case BrowserSync sent requests to http://site.dev:3000 and hotel proxy returned 404 for it.

Now `http://site.dev:3000` should resolve to `http://127.0.0.1:3000` and shouldn't return a 404 anymore.
